src/command_modules/azure-cli-subscription/azure/cli/command_modules/subscription/_help.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+
+helps['subscriptiondefinition'] = """
```

Comment:
This is a very lengthy name for a top-level command. Also, the `account` subgroup  is already designated for subscriptions, so it would be more logical for these command to live there.  `az account definition ...`.

Reply:
Moved it under the account subgroup. Agree the name is long, but unfortunately I believe "definition" is too generic. Is there precedent for using shortened names, such as "subdef", so it becomes "account subdef list"?